web/crux/install-docker.sh.hbr

```diff
@@ -26,6 +33,10 @@ set_environment() {
         echo "Detected: Linux"
         PERSISTENCE_FOLDER=/srv/dagent
         ;;
+      'MINGW64')
+        echo "Detected: Windows(mingw64)"
+        PERSISTENCE_FOLDER="c:/srv/dagent"
```

Comment:
`HOST_DOCKER_SOCK_PATH=//var/run/docker.sock` is not missing?

Reply:
I'll test it but I think too it's missing.

Reply:
That was one of my lost modifications, when I did a backflip in git :see_no_evil: Thanks!